main.tf

```diff
@@ -1 +1,76 @@
+provider "aws" {}
 
+provider "aws" {
+  alias = "master"
+}
+
+data "aws_caller_identity" "member" {
+}
+
+data "aws_caller_identity" "master" {
+  provider = aws.master
+}
+
+data "aws_iam_policy_document" "macie_handshake" {
```

Comment:
any particular reason not to use the standard resource id `this` throughout the module, as none of these resources are duplicated?

Reply:
Fixed